libraries/SITL/SITL.cpp

```diff
@@ -168,10 +178,25 @@ const AP_Param::GroupInfo SIM::var_info[] = {
     // @Description: Opflow data delay
     // @Units: ms
     AP_GROUPINFO("FLOW_DELAY",    36, SIM,  flow_delay, 0),
+    // @Param: ADSB_COUNT
+    // @DisplayName: Number of ADSB aircrafts
+    // @Description: Total number of ADSB simulated aircrafts
```

Comment:
```suggestion
    // @Description: Total number of ADSB simulated aircraft
```

plural of aircraft is... aircraft